packages/clay-nav/src/Vertical.tsx

```diff
@@ -132,7 +132,7 @@ function Item({
 
 			{subItems && (
 				<CSSTransition
-					className={!expanded && 'collapse'}
+					className={expanded ? '' : 'collapse'}
```

Comment:
To avoid rendering with an empty `class`, we can just add an `undefined` value.

```suggestion
					className={expanded ? undefined : 'collapse'}
```

Reply:
Sounds good !